services/consensusalgo/benchmarkconsensus/test/context.go

```diff
@@ -0,0 +1,68 @@
+package test
+
+import (
+	"github.com/orbs-network/go-mock"
+	"github.com/orbs-network/orbs-network-go/config"
+	"github.com/orbs-network/orbs-network-go/instrumentation"
+	"github.com/orbs-network/orbs-network-go/services/consensusalgo/benchmarkconsensus"
+	testInstrumentation "github.com/orbs-network/orbs-network-go/test/harness/instrumentation"
+	"github.com/orbs-network/orbs-spec/types/go/protocol/consensus"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/orbs-network/orbs-spec/types/go/services/gossiptopics"
+)
+
+type context struct {
```

Comment:
let's sit tomorrow morning and refactor this. It's really the wrong direction IMO. Look at the test for Block Storage for what I'd like to do (test driver rather than dumb context): https://github.com/orbs-network/orbs-network-go/blob/master/services/blockstorage/test/commit_block.go

Reply:
I'm letting the context emerge from my tests in TDD :)
I didn't want to create it with a special direction in mind, what you see is the result of refactoring from test to test. I want to see where the next tests take it